nbgrader/html/templates/notebook_submissions.tpl

```diff
@@ -1,8 +1,8 @@
 {%- extends 'gradebook.tpl' -%}
 
 {%- block breadcrumb -%}
-<li><a href="/assignments">Assignments</a></li>
-<li><a href="/assignments/{{ assignment_id }}">{{ assignment_id }}</a></li>
+<li><a href="../assignments">Assignments</a></li>
```

Comment:
Is this jinja? If so, a `{{base_url}}` might be more robust than `..` everywhere.